### DISCUSSION TITLE
Questions about Git submodules support

### DISCUSSION THREAD

**Discussion by ghost** [renovatebot/renovate#29429] [Suggest an Idea]
### Tell us more.

The initial support for git submodules was introduced in November 2019. Based on the documentation, this feature is still beta testing, and the user must opt-in to test it.

As this feature has been available for quite some time now, I have two questions:

1 - Is this feature stable now?
2 - Do you plan to enable it by default?

Thank you.

**Comment by rarkins** [comment]
I think we could remove the experimental tag, but we might still want to keep it disabled by default
